client/src/pages/Home.jsx

```diff
@@ -75,21 +76,23 @@ export default function Home() {
         </DialogActions>
     </Dialog>
 
-    <Grid container justifyContent="center" className={classes.MarginTop}>
-      <Grid item sm={6} xs={12}>
-        <Typography variant="subtitle">Code for Chicago</Typography>
+    <RegPage container justifyContent="center" >
+      <RegBody item sm={6} xs={12}>
+        {/* <Typography variant="subtitle">Code for Chicago</Typography> */}
 
-        <Typography gutterBottom variant="h4" component="h1">Volunteer Registration</Typography>
+        <h1>Volunteer Registration</h1>
 
-        <Typography gutterBottom vairant="body" paragraph>
+        <p gutterBottom variant="body"  color="Blue" paragraph>
```

Comment:
The `<p>` element has a lot of the global styles you need already applied, so you can replace this entire line with just `<p>`